src/agent/src/main.rs

```diff
@@ -382,6 +404,103 @@ async fn start_sandbox(
     Ok(())
 }
 
+// If we fail to start the AA, ocicrypt won't be able to unwrap keys
+// and container decryption will fail.
```

Comment:
This comment isn't really accurate anymore and doesn't describe what this function does.